internal/interpreter/dynamic_component.rs

```diff
@@ -987,6 +987,7 @@ pub(crate) fn generate_component<'id>(
             }
             Type::Struct { .. } => property_info::<Value>(),
             Type::Array(_) => property_info::<Value>(),
+            Type::Easing => property_info::<Value>(),
```

Comment:
I think it would  be better to use the actual underlying type directly

```suggestion
            Type::Easing => property_info::<i_slint_core::animations::EasingCurve>(),
```